go.mod

```diff
@@ -32,14 +34,15 @@ require (
 	golang.org/x/net v0.20.0
 	gopkg.in/k8snetworkplumbingwg/multus-cni.v4 v4.0.2
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.28.7
+	k8s.io/api v0.29.1
```

Comment:
We can't bump to 0.29.1 yet, is it hard requirements ?

Reply:
Fixed